perllib/FixMyStreet/Cobrand/Surrey.pm

```diff
@@ -86,6 +86,24 @@ sub reopening_disallowed { 1 }
 
 sub allow_anonymous_reports { 'button' }
 
+=item * Do not allow email addresses in title or detail
+
+=back
+
+=cut
+
+sub report_validation {
+    my ($self, $report, $errors) = @_;
+
+    my $regex = Utils::email_regex;
+
+    if ($report->detail =~ /$regex/ || $report->title =~ /$regex/) {
+        $errors->{detail} = 'Please remove any email addresses and other personal information from report';
```

Comment:
"information from report" needs rewording

Reply:
> "information from report" needs rewording

Would "information from your report" be OK? Or do you have something else in mind?

Reply:
Yeah, that's fine, it just wasn't English as is :)